app/src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -1,4 +1,41 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 import type {ErrorPageProps} from './types';
+import './error.scss';
 
-export const ErrorPage = ({type}: ErrorPageProps): JSX.Element => <h3>{type}</h3>;
+export const ErrorPage = ({type}: ErrorPageProps): JSX.Element => {
+    let errorTxt = '';
+    switch (type) {
+        case '404':
+            errorTxt = 'Ничего не найдено';
+            break;
+        case '409':
+            errorTxt = 'У вас нет доступа';
+            break;
+        case '500':
+            errorTxt = 'Уппс, у нас произошла ошибка';
+            break;
+        default:
+            errorTxt = 'Что-то пошло не так!';
+    }
+
+    return (
+        <main className="error">
+            <div className="error__inner">
+                <div className="error__header">
+                    <h1 className="error__header-front">{type}</h1>
+                    <span className="error__header-back">{type}</span>
+                </div>
+                <div className="error__cnt">
+                    {errorTxt}
+                    <div className="mt-2" />
+                    Мы скоро всё исправим. <br />А пока можно начать с самого начала.
+                    <div className="mt-5" />
+                    <Link to="/" className="link">
+                        На старт
```

Comment:
В приложении есть Button. Думаю стоит его впилить вместо текста. Ну или обернуть текст в span
```suggestion
                        <span>На старт</span>
```